### DISCUSSION TITLE
Droplet requirements for 4 station install

### DISCUSSION THREAD

**Discussion by darrylcarter** [AzuraCast/AzuraCast#4180] [Q&A]
I've installed the droplet on digital ocean to the minimum requirements

Can I realistically expect it to handle 4 stations. I'm thinking of relaying the listener load to relays or is that not an issue? Each station will be a third live two thirds autodj

**Using Docker installation method?**

**AzuraCast version:**

**Host Operating System:**

**Describe the bug:**

**Steps to reproduce:**

**Expected behavior:**

**Relevant logs:**

**Screenshots:**

**Device(s):**

- Device: 
- OS: 
- Browser: 
- Version: 

**Additional context:**

**Comment by ItsMitchh** [comment]
Generally, the minimum specs means that the software may work out the box, but we cannot guarantee it'll work with many stations, it'll boot with at least one.
